tests/e2e/Pages/AggregatesComponet.ts

```diff
@@ -73,6 +85,43 @@ class AggregatesComponent extends DashboardPage {
       await expect(layerLocator).toBeVisible();
     }
   }
+
+  async validatesAggregatesInfoButtons() {
+    // click on the first info icon and validate the opopver content
+    await this.aggreagtesTitleInfoIcon.click();
+    const disclaimerText = await this.approximatedisclaimer.textContent();
+    expect(disclaimerText).toContain(
+      'All numbers are approximate and meant to be used as guidance.',
```

Comment:
I'd move these strings to a helper file. Design may decide to change these strings. I'd prefer if these strings are separated from the test logic.

Reply:
Fixed this and another one in different PR: https://github.com/rodekruis/IBF-system/pull/1658